### ISSUE TITLE
Deprecate ReSpec style sheet 

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/respec#748]
I wonder if we can deprecate it and fold things into the w3c styles. Having multiple styles seems redundant.

**Comment by halindrome** [comment]
Actually, the APA working group has some concerns about our styles. Apparently they don't confirm to the WCAG contrast rules or something.  If we push this stuff into the master sheets then 1) all specs can look the same, and 2) it will be someone else's problem!  @fantasai or @plehegar are you open to adding styles for notes, warnings, issues, etc. into the master styles?

**Comment by marcoscaceres** [comment]
@halindrome, I think they already support issues, notes, etc. in the W3C stylesheet. I was doing a quick review last night of the overlap and there is quite a bit. We might need to keep a few little things, but maybe about 80% of the ReSpec stylesheets can probably be removed.

**Comment by halindrome** [comment]
Ok.  So presumably all we need do is ensure we are using the right class names.  I can take a look at the style sheets again too. 

What about IDL formatting?

**Comment by marcoscaceres** [comment]
> What about IDL formatting?

We should probably coordinate with @tabatkins on making sure syntax highlighting is the same.

**Comment by halindrome** [comment]
Yeah - pretty sure it is not currently.

**Comment by tabatkins** [comment]
I'm not currently syntax-highlighting IDL at all; I'd like to do so, tho!

(Both the IDL parser/linkifier and the syntax highlighter take raw text and output markup, so they can't be combined. Instead I'll probably host IDL syntax highlighting on top of the output I get from the IDL parser.)

**Comment by fantasai** [comment]
The W3C stylesheets currently don't include any syntax highlighting; that's intentionally left for further experimentation by the WGs/editors/preprocessors. Most of the boxes should be covered already
  http://fantasai.inkedblade.net/style/design/w3c-restyle/2016/sample#boxes
(IDL definitions should in most cases be using class='def'.)

Let me know if there's anything else missing?

**Comment by marcoscaceres** [comment]
We’ve removed what we can from ReSpec’s style sheet.
